Tests/SwiftDocCTests/Diagnostics/DiagnosticTests.swift

```diff
@@ -140,6 +141,65 @@ class DiagnosticTests: XCTestCase {
         \(expectedNoteLocation): note: The message of the note.
         """)
     }
+
+    func testDoxygenDiagnostic() throws {
+        let identifer = SymbolGraph.Symbol.Identifier(precise: "s:5MyKit0A5ClassC10myFunctionyyF", interfaceLanguage: "occ")
+        let names = SymbolGraph.Symbol.Names(title: "", navigator: nil, subHeading: nil, prose: nil)
+        let pathComponents = ["path", "to", "my file.h"]
+        let range = SymbolGraph.LineList.SourceRange(
+            start: .init(line: 0, character: 0),
+            end: .init(line: 0, character: 0)
+        )
+        let commentText = """
+          /// Brief description of this method
+          @param something Description of this parameter
+          @returns Description of return value
+          """
+        let lines = commentText.components(separatedBy: .newlines).map { SymbolGraph.LineList.Line(text: $0, range: range) }
+        let docComment = SymbolGraph.LineList(lines)
+
+        let symbol = SymbolGraph.Symbol(
```

Comment:
I would find it easier to read this test and identify what's different about the two symbols if the were created in a common code path where the information that varies was passed as arguments. For example a test helper like this:
```swift
func createTestSymbol(commentText: String) -> SymbolGraph.Symbol {
     let emptyRange = SymbolGraph.LineList.SourceRange(
         start: .init(line: 0, character: 0),
         end: .init(line: 0, character: 0)
     )
    let docCommentLines = commentText.components(separatedBy: .newlines).map { SymbolGraph.LineList.Line(text: $0, range: emptyRange) }
    return SymbolGraph.Symbol(
        identifier: .init(precise: "s:5MyKit0A5ClassC10myFunctionyyF", interfaceLanguage: "occ"),
        names: .init(title: "", navigator: nil, subHeading: nil, prose: nil),
        pathComponents: ["path", "to", "my file.h"],
        docComment: SymbolGraph.LineList(docCommentLines),
        accessLevel: .init(rawValue: "public"),
        kind: .init(parsedIdentifier: .func, displayName: "myFunction"),
        mixins: [:]
    )
}
```